chat/src/main/java/net/md_5/bungee/api/chat/ComponentBuilder.java

```diff
@@ -77,8 +112,9 @@ public ComponentBuilder append(String text, FormatRetention retention)
     {
         parts.add( current );
 
```

Comment:
This is because the 'current' variable was changed to a BaseComponent instead of a TextComponent and add BaseComponent doesn't have a setText method, it won't compile. I could of course just cast it instead of using a new variable.